File: rnd_number/GoLang/rnd.go
/* Use with go run rnd.go OR go build rnd.go for compiling */
// Generates and prints a random number from 0 to 1000
package main

import ( "fmt"; "math/rand"; "time")

func main(){

    seed := rand.NewSource(time.Now().UnixNano())
    randNr := rand.New(seed)
    fmt.Println("Your random number is: ", randNr.Intn(1000))
}

File: rnd_number/Rust/rnd.rs
/* Compile using rustc rnd.rs */
// Generates and prints a random number from 0 to 1000

//extern crate rand;
use std::rand;
use std::rand::Rng;

fn main() {
    let mut rng = rand::thread_rng();

    println!("Here is your random number: {}", rng.gen_range(0..1001));

}
